clvm-utils/src/curried_program.rs

```diff
@@ -5,7 +5,7 @@ use clvm_traits::{
 use clvmr::{allocator::NodePtr, Allocator};
 
 #[derive(Debug, Clone)]
-pub struct CurriedProgram<T>(NodePtr, T);
+pub struct CurriedProgram<T>(pub NodePtr, pub T);
```

Comment:
in fact, I think these fields would benefit from being named. e.g. `puzzle` and `args`. or maybe `program` and `args`